src/Schema/CMS/Events/BulkEditFlow.ts

```diff
@@ -195,14 +197,16 @@ export interface CmsBulkEditProcessingStarted {
  * {
  *   action: "processingCompleted",
  *   context_module: "Artworks - bulk edit",
- *   value: 24  // total number of artworks successfully processed
+ *   label: "change availability",
+ *   value: "on hold", // e.g. "on hold", "available", "not for sale"
```

Comment:
`label` and `value` will change based on the three options we have (so far)

Label can be:
```
- change availability
- change location
- change medium
```

and then value will be any of the selected options for the field they are editing
```
Ex:
- An availability: "on hold"
- An address: "123 Happy St, NY NY 10009"
- A medium: "Painting"
```